### ISSUE TITLE
Incompatability with Graph Lib / PolyFactory

### ISSUE THREAD

**Issue by CamperSamu** [CesiumMC/cesium-fabric#3]
The server doesn't actually crash, but it fails to load the world.

Minecraft: 1.20.4
Cesium: [0.1.7](https://modrinth.com/mod/cesium/version/Vl4sGn0W)
PolyFactory: [0.3.0](https://modrinth.com/mod/polyfactory/version/0.3.0+1.20.4)
GraphLib: [1.4.0](https://github.com/Kneelawk/GraphLib/releases/tag/v1.4.0%2B1.20.4)

#### Stacktrace
```apache
Failed to read chunk [0, 0]
java.lang.NullPointerException: Cannot invoke "de.yamayaki.cesium.common.db.LMDBInstance.getDatabase(de.yamayaki.cesium.common.db.spec.DatabaseSpec)" because "this.lmdbStorage" is null
	at net.minecraft.world.storage.StorageIoWorker.mddc692b$cesium$lambda$loadAsync$0$4(StorageIoWorker.java:1073) ~[server-intermediary.jar:?]
	at net.minecraft.world.storage.StorageIoWorker.method_27939(StorageIoWorker.java:236) ~[server-intermediary.jar:?]
	at net.minecraft.util.thread.TaskQueue$PrioritizedTask.run(TaskQueue.java:60) ~[server-intermediary.jar:?]
	at net.minecraft.util.thread.TaskExecutor.runNext(TaskExecutor.java:91) ~[server-intermediary.jar:?]
	at net.minecraft.util.thread.TaskExecutor.runWhile(TaskExecutor.java:146) ~[server-intermediary.jar:?]
	at net.minecraft.util.thread.TaskExecutor.run(TaskExecutor.java:102) ~[server-intermediary.jar:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642) ~[?:?]
	at java.lang.Thread.run(Thread.java:1583) ~[?:?]
Failed to read chunk [0, 0]
java.lang.NullPointerException: Cannot invoke "de.yamayaki.cesium.common.db.LMDBInstance.getDatabase(de.yamayaki.cesium.common.db.spec.DatabaseSpec)" because "this.lmdbStorage" is null
	at net.minecraft.world.storage.StorageIoWorker.mddc692b$cesium$lambda$loadAsync$0$4(StorageIoWorker.java:1073) ~[server-intermediary.jar:?]
	at net.minecraft.world.storage.StorageIoWorker.method_27939(StorageIoWorker.java:236) ~[server-intermediary.jar:?]
	at net.minecraft.util.thread.TaskQueue$PrioritizedTask.run(TaskQueue.java:60) ~[server-intermediary.jar:?]
	at net.minecraft.util.thread.TaskExecutor.runNext(TaskExecutor.java:91) ~[server-intermediary.jar:?]
	at net.minecraft.util.thread.TaskExecutor.runWhile(TaskExecutor.java:146) ~[server-intermediary.jar:?]
	at net.minecraft.util.thread.TaskExecutor.run(TaskExecutor.java:102) ~[server-intermediary.jar:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642) ~[?:?]
	at java.lang.Thread.run(Thread.java:1583) ~[?:?]
Error loading chunk pillar [0, 0].
java.util.concurrent.CompletionException: java.lang.NullPointerException: Cannot invoke "de.yamayaki.cesium.common.db.LMDBInstance.getDatabase(de.yamayaki.cesium.common.db.spec.DatabaseSpec)" because "this.lmdbStorage" is null
	at java.util.concurrent.CompletableFuture.reportJoin(CompletableFuture.java:413) ~[?:?]
	at java.util.concurrent.CompletableFuture.join(CompletableFuture.java:2118) ~[?:?]
	at com.kneelawk.graphlib.api.world.UnloadingRegionBasedStorage.getIfExists(UnloadingRegionBasedStorage.java:166) ~[graphlib-1.4.0+1.20.4-4df9c0650fb779a5.jar:?]
	at com.kneelawk.graphlib.impl.graph.simple.SimpleServerGraphWorld.loadGraphs(SimpleServerGraphWorld.java:1450) ~[graphlib-1.4.0+1.20.4-4df9c0650fb779a5.jar:?]
	at com.kneelawk.graphlib.impl.graph.simple.SimpleServerGraphWorld.onWorldChunkLoad(SimpleServerGraphWorld.java:162) ~[graphlib-1.4.0+1.20.4-4df9c0650fb779a5.jar:?]
	at com.kneelawk.graphlib.impl.graph.ServerGraphWorldStorage.onWorldChunkLoad(ServerGraphWorldStorage.java:62) ~[graphlib-1.4.0+1.20.4-4df9c0650fb779a5.jar:?]
	at com.kneelawk.graphlib.impl.GraphLibFabricMod.lambda$onInitialize$1(GraphLibFabricMod.java:33) ~[graphlib-1.4.0+1.20.4-4df9c0650fb779a5.jar:?]
	at net.fabricmc.fabric.api.event.lifecycle.v1.ServerChunkEvents.lambda$static$0(ServerChunkEvents.java:36) ~[fabric-lifecycle-events-v1-2.2.2+78d798af4f-b94fc22ba1a9e0f4.jar:?]
	at net.minecraft.server.world.ThreadedAnvilChunkStorage.handler$bed000$fabric-lifecycle-events-v1$onChunkLoad(ThreadedAnvilChunkStorage.java:5951) ~[server-intermediary.jar:?]
	at net.minecraft.server.world.ThreadedAnvilChunkStorage.method_17227(ThreadedAnvilChunkStorage.java:744) ~[server-intermediary.jar:?]
	at com.mojang.datafixers.util.Either.lambda$mapLeft$0(Either.java:162) ~[datafixerupper-6.0.8.jar:?]
	at com.mojang.datafixers.util.Either$Left.map(Either.java:38) ~[datafixerupper-6.0.8.jar:?]
	at com.mojang.datafixers.util.Either.mapLeft(Either.java:162) ~[datafixerupper-6.0.8.jar:?]
	at net.minecraft.server.world.ThreadedAnvilChunkStorage.method_20460(ThreadedAnvilChunkStorage.java:726) ~[server-intermediary.jar:?]
	at java.util.concurrent.CompletableFuture$UniApply.tryFire(CompletableFuture.java:646) ~[?:?]
	at java.util.concurrent.CompletableFuture$Completion.run(CompletableFuture.java:482) ~[?:?]
	at net.minecraft.util.thread.ThreadExecutor.executeTask(ThreadExecutor.java:156) ~[server-intermediary.jar:?]
	at net.minecraft.server.world.ServerChunkManager$MainThreadExecutor.executeTask(ServerChunkManager.java:561) ~[server-intermediary.jar:?]
	at net.minecraft.util.thread.ThreadExecutor.runTask(ThreadExecutor.java:130) ~[server-intermediary.jar:?]
	at net.minecraft.server.world.ServerChunkManager$MainThreadExecutor.runTask(ServerChunkManager.java:570) ~[server-intermediary.jar:?]
	at net.minecraft.util.thread.ThreadExecutor.runTasks(ThreadExecutor.java:139) ~[server-intermediary.jar:?]
	at net.minecraft.server.world.ServerChunkManager.redirect$zbm000$c2me-base$beforeAwaitChunk(ServerChunkManager.java:4636) ~[server-intermediary.jar:?]
	at net.minecraft.server.world.ServerChunkManager.getChunk(ServerChunkManager.java:138) ~[server-intermediary.jar:?]
	at net.minecraft.world.World.getChunk(World.java:207) ~[server-intermediary.jar:?]
	at net.minecraft.world.WorldView.getChunk(WorldView.java:156) ~[server-intermediary.jar:?]
	at net.minecraft.world.World.getChunk(World.java:201) ~[server-intermediary.jar:?]
	at net.minecraft.server.network.SpawnLocating.findOverworldSpawn(SpawnLocating.java:23) ~[server-intermediary.jar:?]
	at net.minecraft.server.network.SpawnLocating.findServerSpawnPoint(SpawnLocating.java:64) ~[server-intermediary.jar:?]
	at net.minecraft.server.MinecraftServer.setupSpawn(MinecraftServer.java:442) ~[server-intermediary.jar:?]
	at net.minecraft.server.MinecraftServer.createWorlds(MinecraftServer.java:374) ~[server-intermediary.jar:?]
	at net.minecraft.server.MinecraftServer.loadWorld(MinecraftServer.java:330) ~[server-intermediary.jar:?]
	at net.minecraft.server.dedicated.MinecraftDedicatedServer.setupServer(MinecraftDedicatedServer.java:169) ~[server-intermediary.jar:?]
	at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:650) ~[server-intermediary.jar:?]
	at net.minecraft.server.MinecraftServer.method_29739(MinecraftServer.java:270) ~[server-intermediary.jar:?]
	at java.lang.Thread.run(Thread.java:1583) ~[?:?]
Caused by: java.lang.NullPointerException: Cannot invoke "de.yamayaki.cesium.common.db.LMDBInstance.getDatabase(de.yamayaki.cesium.common.db.spec.DatabaseSpec)" because "this.lmdbStorage" is null
	at net.minecraft.world.storage.StorageIoWorker.mddc692b$cesium$lambda$loadAsync$0$4(StorageIoWorker.java:1073) ~[server-intermediary.jar:?]
	at net.minecraft.world.storage.StorageIoWorker.method_27939(StorageIoWorker.java:236) ~[server-intermediary.jar:?]
	at net.minecraft.util.thread.TaskQueue$PrioritizedTask.run(TaskQueue.java:60) ~[server-intermediary.jar:?]
	at net.minecraft.util.thread.TaskExecutor.runNext(TaskExecutor.java:91) ~[server-intermediary.jar:?]
	at net.minecraft.util.thread.TaskExecutor.runWhile(TaskExecutor.java:146) ~[server-intermediary.jar:?]
	at net.minecraft.util.thread.TaskExecutor.run(TaskExecutor.java:102) ~[server-intermediary.jar:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642) ~[?:?]
	... 1 more
```

**Comment by yamayaki1** [comment]
The latest build should fix the incompatibility. You can download the build through [GitHub Actions](https://github.com/yamayaki1/cesium-fabric/actions/runs/7213626115) or by building [this commit](https://github.com/yamayaki1/cesium-fabric/commit/e3c0b5eda1ad84e2a4a53a33205f4f60d6f212c6) yourself.

**Comment by CamperSamu** [comment]
Thanks for the heads up!
I will try and play with it later today.

**Comment by CamperSamu** [comment]
I can confirm the fix to be working!

No corruption unpon saving and loading.

Thanks for your work!

![image](https://github.com/yamayaki1/cesium-fabric/assets/32773961/e15b05ad-1e01-4773-98c9-3be21797a117)

**Comment by CamperSamu** [comment]
In case you are curious, this is the modlist I am testing Cesium against:
```apache
- alternate-current 1.7.0
- antixray 1.3.1
   \-- com_moandjiezana_toml_toml4j 0.7.2
- armor-stand-editor 2.3.0+1.20.3-rc1
   |-- common-protection-api 1.0.0
   |-- player-data-api 0.4.0+1.20.3
   |-- server_translations_api 2.2.0+1.20.3-rc1
   |    \-- packet_tweaker 0.5.0+1.20.2-rc1
   \-- sgui 1.3.0+1.20.2
- balm-fabric 8.0.2
- blossom-homes 2.2.2+1.20.4
- blossom-lib 2.5.6+1.20.4
   \-- server_translations_api 2.2.0+1.20.3-rc1
        \-- packet_tweaker 0.5.0+1.20.2-rc1
- blossom-tpa 2.2.2+1.20.4
- blossom-warps 2.0.6+1.20.4
- c2me 0.2.0+alpha.11.38
   |-- c2me-base 0.2.0+alpha.11.38
   |-- c2me-client-uncapvd 0.2.0+alpha.11.38
   |-- c2me-fixes-chunkio-threading-issues 0.2.0+alpha.11.38
   |-- c2me-fixes-general-threading-issues 0.2.0+alpha.11.38
   |-- c2me-fixes-worldgen-threading-issues 0.2.0+alpha.11.38
   |-- c2me-fixes-worldgen-vanilla-bugs 0.2.0+alpha.11.38
   |-- c2me-notickvd 0.2.0+alpha.11.38
   |-- c2me-opts-allocs 0.2.0+alpha.11.38
   |-- c2me-opts-chunk-access 0.2.0+alpha.11.38
   |-- c2me-opts-chunkio 0.2.0+alpha.11.38
   |-- c2me-opts-math 0.2.0+alpha.11.38
   |-- c2me-opts-scheduling 0.2.0+alpha.11.38
   |-- c2me-opts-worldgen-general 0.2.0+alpha.11.38
   |-- c2me-opts-worldgen-vanilla 0.2.0+alpha.11.38
   |-- c2me-server-utils 0.2.0+alpha.11.38
   |-- c2me-threading-chunkio 0.2.0+alpha.11.38
   |-- c2me-threading-lighting 0.2.0+alpha.11.38
   |-- c2me-threading-worldgen 0.2.0+alpha.11.38
   |-- com_ibm_async_asyncutil 0.1.0
   |-- net_objecthunter_exp4j 0.4.8
   \-- org_threadly_threadly 7.0
- carpet 1.4.128+v231205
- cesium 0.1.8
   |-- com_github_jnr_jffi 1.3.12
   |-- com_github_jnr_jffi_native 1.3.12
   |-- com_github_jnr_jnr-a64asm 1.0.0
   |-- com_github_jnr_jnr-constants 0.10.4
   |-- com_github_jnr_jnr-ffi 2.2.15
   |-- com_github_jnr_jnr-x86asm 1.0.2
   |-- com_github_luben_zstd-jni 1.5.5-11
   \-- org_lmdbjava_lmdbjava 0.9.0
- chqoneblock 1.0.0
   |-- blue_endless_jankson 1.2.3
   |-- com_squareup_moshi_moshi 1.15.0
   |-- com_squareup_okio_okio-jvm 3.6.0
   |-- common-economy-api 1.1.1
   |-- common-protection-api 1.0.0
   |-- dev_zacsweers_moshix_moshi-records-reflect 0.15.0-SNAPSHOT
   |-- factorytools 0.1.0+1.20.4
   |-- hologram-api 0.2.5+1.20.2
   |-- placeholder-api 2.3.0+1.20.3
   |-- polymer-blocks 0.7.1+1.20.4
   |-- polymer-core 0.7.1+1.20.4
   |    |-- polymer-networking 0.7.1+1.20.4
   |    |    \-- polymer-common 0.7.1+1.20.4
   |    |         \-- packet_tweaker 0.5.0+1.20.2-rc1
   |    \-- polymer-registry-sync-manipulator 0.7.1+1.20.4
   |-- polymer-networking 0.7.1+1.20.4
   |    \-- polymer-common 0.7.1+1.20.4
   |         \-- packet_tweaker 0.5.0+1.20.2-rc1
   |-- polymer-resource-pack 0.7.1+1.20.4
   |    \-- polymer-common 0.7.1+1.20.4
   |         \-- packet_tweaker 0.5.0+1.20.2-rc1
   |-- polymer-virtual-entity 0.7.1+1.20.4
   |    \-- polymer-common 0.7.1+1.20.4
   |         \-- packet_tweaker 0.5.0+1.20.2-rc1
   |-- server_translations_api 2.2.0+1.20.3-rc1
   |    \-- packet_tweaker 0.5.0+1.20.2-rc1
   |-- sgui 1.3.0+1.20.2
   \-- sidebar-api 0.3.0+1.20.3
- cloth-config 13.0.114
   \-- cloth-basic-math 0.6.1
- clumps 13.0.0.4
- common-bridge 0.1.0
   |-- common-economy-api 1.1.1
   \-- common-protection-api 1.0.0
- debugify 1.20.3+1.1
- entity-view-distance 1.2.0+1.20.4
   \-- spruceui 5.0.3+1.20.4
- fabric-api 0.91.2+1.20.4
   |-- fabric-api-base 0.4.35+78d798af4f
   |-- fabric-api-lookup-api-v1 1.6.45+78d798af4f
   |-- fabric-biome-api-v1 13.0.15+78d798af4f
   |-- fabric-block-api-v1 1.0.14+78d798af4f
   |-- fabric-block-view-api-v2 1.0.3+78d798af4f
   |-- fabric-blockrenderlayer-v1 1.1.45+78d798af4f
   |-- fabric-client-tags-api-v1 1.1.6+78d798af4f
   |-- fabric-command-api-v1 1.2.40+f71b366f4f
   |-- fabric-command-api-v2 2.2.19+78d798af4f
   |-- fabric-commands-v0 0.2.57+df3654b34f
   |-- fabric-containers-v0 0.1.81+df3654b34f
   |-- fabric-content-registries-v0 5.0.9+78d798af4f
   |-- fabric-convention-tags-v1 1.5.9+78d798af4f
   |-- fabric-crash-report-info-v1 0.2.22+78d798af4f
   |-- fabric-data-generation-api-v1 13.1.17+78d798af4f
   |-- fabric-dimensions-v1 2.1.60+78d798af4f
   |-- fabric-entity-events-v1 1.5.27+78d798af4f
   |-- fabric-events-interaction-v0 0.6.14+78d798af4f
   |-- fabric-events-lifecycle-v0 0.2.71+df3654b34f
   |-- fabric-game-rule-api-v1 1.0.45+78d798af4f
   |-- fabric-item-api-v1 2.1.34+78d798af4f
   |-- fabric-item-group-api-v1 4.0.20+78d798af4f
   |-- fabric-key-binding-api-v1 1.0.40+78d798af4f
   |-- fabric-keybindings-v0 0.2.38+df3654b34f
   |-- fabric-lifecycle-events-v1 2.2.29+78d798af4f
   |-- fabric-loot-api-v2 2.1.4+78d798af4f
   |-- fabric-message-api-v1 6.0.4+78d798af4f
   |-- fabric-mining-level-api-v1 2.1.59+78d798af4f
   |-- fabric-model-loading-api-v1 1.0.7+78d798af4f
   |-- fabric-models-v0 0.4.6+9386d8a74f
   |-- fabric-networking-api-v1 3.1.2+78d798af4f
   |-- fabric-object-builder-api-v1 13.0.7+78d798af4f
   |-- fabric-particles-v1 1.1.6+78d798af4f
   |-- fabric-recipe-api-v1 2.0.15+78d798af4f
   |-- fabric-registry-sync-v0 4.0.13+78d798af4f
   |-- fabric-renderer-api-v1 3.2.3+78d798af4f
   |-- fabric-renderer-indigo 1.5.3+78d798af4f
   |-- fabric-renderer-registries-v1 3.2.50+df3654b34f
   |-- fabric-rendering-data-attachment-v1 0.3.41+73761d2e4f
   |-- fabric-rendering-fluids-v1 3.0.32+78d798af4f
   |-- fabric-rendering-v0 1.1.53+df3654b34f
   |-- fabric-rendering-v1 3.0.12+78d798af4f
   |-- fabric-resource-conditions-api-v1 2.3.13+78d798af4f
   |-- fabric-resource-loader-v0 0.11.15+78d798af4f
   |-- fabric-screen-api-v1 2.0.16+78d798af4f
   |-- fabric-screen-handler-api-v1 1.3.50+78d798af4f
   |-- fabric-sound-api-v1 1.0.16+78d798af4f
   |-- fabric-transfer-api-v1 4.0.6+78d798af4f
   \-- fabric-transitive-access-wideners-v1 5.0.13+78d798af4f
- fabric-language-kotlin 1.10.16+kotlin.1.9.21
   |-- org_jetbrains_kotlin_kotlin-reflect 1.9.21
   |-- org_jetbrains_kotlin_kotlin-stdlib 1.9.21
   |-- org_jetbrains_kotlin_kotlin-stdlib-jdk7 1.9.21
   |-- org_jetbrains_kotlin_kotlin-stdlib-jdk8 1.9.21
   |-- org_jetbrains_kotlinx_atomicfu-jvm 0.23.1
   |-- org_jetbrains_kotlinx_kotlinx-coroutines-core-jvm 1.7.3
   |-- org_jetbrains_kotlinx_kotlinx-coroutines-jdk8 1.7.3
   |-- org_jetbrains_kotlinx_kotlinx-datetime-jvm 0.4.1
   |-- org_jetbrains_kotlinx_kotlinx-serialization-cbor-jvm 1.6.2
   |-- org_jetbrains_kotlinx_kotlinx-serialization-core-jvm 1.6.2
   \-- org_jetbrains_kotlinx_kotlinx-serialization-json-jvm 1.6.2
- fabricloader 0.15.1
   \-- mixinextras 0.3.1
- fabrictailor 2.3.0
   \-- config2brigadier 1.2.3
- fantasy 0.5.0+1.20.4
- ferritecore 6.0.2
- globalpacks 1.16.5
- holograms 1.5.2+1.20.3-legacy-edition
   |-- hologram-api 0.2.5+1.20.2
   |-- placeholder-api 2.3.0+1.20.3
   \-- server_translations_api 2.2.0+1.20.3-rc1
        \-- packet_tweaker 0.5.0+1.20.2-rc1
- image2map 0.5.0+1.20.2
   |-- map-canvas-api 0.2.2+1.19.4
   \-- sgui 1.3.0+1.20.2
- inventorysorter 1.9.0-1.20
   \-- kyrptconfig 1.5.6-1.20
- java 21
- jline4mcdsrv 0.4.3
   |-- org_fusesource_jansi_jansi 2.4.0
   |-- org_jline_jline 3.23.0
   \-- org_jline_jline-terminal-jansi 3.23.0
- kleeslabs 16.0.0
- krypton 0.2.4
   \-- com_velocitypowered_velocity-native 3.2.0-SNAPSHOT
- lithium 0.12.0+git.7b624db
- luckperms 5.4.113
- memoryleakfix 1.1.2
- minecraft 1.20.4
- mixintrace 1.1.1+1.17
- modernfix 5.10.1+mc1.20.4
- nochatreports 1.20.4-v2.5.0
   |-- fabric-rendering-v1 3.0.11+9468a19d32
   \-- fabric-screen-api-v1 2.0.15+f6e85ac232
- nylon 0.9.12
- owo 0.12.0-pre.3+1.20.3
- polyconfig 0.3.7
   \-- dev_hbeck_kdl_kdl4j 0.2.0
- polydex 1.0.0+1.20.3-rc1
   |-- placeholder-api 2.3.0+1.20.3
   |-- player-data-api 0.4.0+1.20.3
   |-- predicate-api 0.3.0+1.20.2
   |-- server_translations_api 2.2.0+1.20.3-rc1
   |    \-- packet_tweaker 0.5.0+1.20.2-rc1
   |-- sgui 1.3.0+1.20.2
   \-- sidebar-api 0.3.0+1.20.3
- polyfactory 0.3.0+1.20.4
   |-- common-protection-api 1.0.0
   |-- factorytools 0.1.0+1.20.4
   |-- graphlib 1.4.0+1.20.4
   |    |-- kmodlib-overlay 0.2.13+1.20.3
   |    \-- libnetworkstack 0.11.0-pre.1
   |-- map-canvas-api 0.2.2+1.19.4
   |-- sgui 1.3.0+1.20.2
   \-- sidebar-api 0.3.0+1.20.3
- polymc 5.5.2+1.20.4
   |-- packet_tweaker 0.5.0+1.20.2-rc1
   \-- resource-locator-api 0.5.0+1.20.1
- polymer-bundled 0.7.1+1.20.4
   |-- polymer-autohost 0.7.1+1.20.4
   |    \-- io_netty_netty-codec-http 4.1.82.Final
   |-- polymer-blocks 0.7.1+1.20.4
   |-- polymer-common 0.7.1+1.20.4
   |    \-- packet_tweaker 0.5.0+1.20.2-rc1
   |-- polymer-core 0.7.1+1.20.4
   |    |-- polymer-networking 0.7.1+1.20.4
   |    |    \-- polymer-common 0.7.1+1.20.4
   |    |         \-- packet_tweaker 0.5.0+1.20.2-rc1
   |    \-- polymer-registry-sync-manipulator 0.7.1+1.20.4
   |-- polymer-resource-pack 0.7.1+1.20.4
   |    \-- polymer-common 0.7.1+1.20.4
   |         \-- packet_tweaker 0.5.0+1.20.2-rc1
   \-- polymer-virtual-entity 0.7.1+1.20.4
        \-- polymer-common 0.7.1+1.20.4
             \-- packet_tweaker 0.5.0+1.20.2-rc1
- raknetify 0.1.0+alpha.5.114
- semblance 1.1.5
- servercore 1.4.0+1.20.3
   |-- com_electronwill_night-config_core 3.6.6
   |-- com_electronwill_night-config_toml 3.6.6
   |-- fabric-permissions-api-v0 0.3-SNAPSHOT
   \-- placeholder-api 2.3.0+1.20.3
- servertick 1.9
- skylands 0.4.0+1.20.4
   |-- cardinal-components-base 5.4.0
   |-- cardinal-components-entity 5.4.0
   |-- cardinal-components-world 5.4.0
   |-- placeholder-api 2.3.0+1.20.3
   \-- server_translations_api 2.2.0+1.20.3-rc1
        \-- packet_tweaker 0.5.0+1.20.2-rc1
- slumber 1.1
- smartbrainlib 1.12
- spark 1.10.54
- stackdeobfuscator 1.4.2+bf1fc22
   |-- dev_booky_common_ 1.4.2+bf1fc22
   \-- net_fabricmc_mapping-io 0.3.0
- starlight 1.1.3+fabric.f5dcd1a
- styled-nicknames 1.4.0+1.20.3
   |-- placeholder-api 2.3.0+1.20.3
   \-- player-data-api 0.4.0+1.20.3
- styled-sidebars 1.3.0+1.20.3
   |-- placeholder-api 2.3.0+1.20.3
   |-- player-data-api 0.4.0+1.20.3
   |-- predicate-api 0.3.0+1.20.2
   \-- sidebar-api 0.3.0+1.20.3
- styledchat 2.4.0+1.20.3
   |-- placeholder-api 2.3.0+1.20.3
   |-- player-data-api 0.4.0+1.20.3
   \-- predicate-api 0.3.0+1.20.2
- styledplayerlist 3.3.0+1.20.3
   |-- placeholder-api 2.3.0+1.20.3
   |-- player-data-api 0.4.0+1.20.3
   \-- predicate-api 0.3.0+1.20.2
- threadtweak 1.20.4-0.1.2
- tpshud 1.7.0+1.20.3
   \-- me_obsilabor_alert 1.0.8
- trinkets 3.8.0+polymerport.1
   \-- sgui 1.3.0+1.20.2
- vanilla-permissions 0.2.2+1.20.2
   \-- fabric-permissions-api-v0 0.3-SNAPSHOT
- viabackwards 4.9.2-SNAPSHOT
- viafabric 0.4.12+52-main
   |-- cotton-client-commands 1.1.0+1.15.2
   |-- org_yaml_snakeyaml 2.2
   \-- viafabric-mc120 0.4.12+52-main
- viarewind 3.0.5
- viaversion 4.9.3-SNAPSHOT
- vmp 0.2.0+beta.7.135
   \-- com_ibm_async_asyncutil 0.1.0
- yet_another_config_lib_v3 3.3.1+1.20.4
   |-- com_twelvemonkeys_common_common-image 3.10.0
   |-- com_twelvemonkeys_common_common-io 3.10.0
   |-- com_twelvemonkeys_common_common-lang 3.10.0
   |-- com_twelvemonkeys_imageio_imageio-core 3.10.0
   |-- com_twelvemonkeys_imageio_imageio-metadata 3.10.0
   |-- com_twelvemonkeys_imageio_imageio-webp 3.10.0
   |-- org_quiltmc_parsers_gson 0.2.1
   \-- org_quiltmc_parsers_json 0.2.1
```
